src/literals.h

```diff
@@ -5,46 +5,79 @@ namespace asyncsrv {
 static constexpr const char* empty = "";
 
 #ifndef ESP8622
```

Comment:
Isn't here a typo? (ESP8266 instead of ESP8266)

Reply:
OMG! @vortigont : this bug is there since a while! That might be why ESP8266 users have some memory issues ?

Reply:
Wow! Nice catch! :)